src/tx/builder/schema.js

```diff
@@ -22,7 +22,8 @@ export const RESPONSE_TTL = { type: 'delta', value: 10 }
 // # CONTRACT
 export const DEPOSIT = 0
 export const AMOUNT = 0
-export const GAS = 25000
+export const GAS_ABOVE_AVERAGE = 25000 // TODO: don't use this
```

Comment:
why do we need this?

Reply:
`gaAttachTx` still depends on it, I would like to deal with it later

Reply:
mhh, I think I get your point. you need this for transactions that do not support dry-runs, correct? I just consulted with @hanssv and got confirmed that the only tx-types that aren't supported by the dry-run endpoint are:
- `GaMetaTx`
- `PayingForTx`

for these transactions we can use the default `25000` in my opinion. I would also remove the `TODO:` wording in the comment. or do you see any other solution? I currently don't 😅

Reply:
We should maybe make gas a required parameter for those then, as otherwise it will be unclear why executions fail?

Reply:
The same estimation as for contract deployment seems to work fine, so I'm removing this constant.

Reply:
is this a problem with the force pushs or how can I see what changed in regards to our comments? I am really struggling to understand what we are doing now.

I understand you removed `GAS_ABOVE_AVERAGE` as you can reuse another estimation.

sorry to ask / comment again:
- `GaAttachTx` can estimated via dry-run according to documentation and Hans => do we do that now? I don't understand why we need to use the logic for contract deployment here 🤔 
- how are we dealing now for `GaMetaTx` and `PayingForTx` ? what amount of `GAS` do we use there per default? should we open another ticket for this?